scripts/pugme.js

```diff
@@ -0,0 +1,41 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, msg => msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => msg.send(JSON.parse(body).pug)));
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
+    let count;
+    if (parseInt(msg.match[2]) > 10) {
+      count = 10;
+    }
+    else {
+      count = msg.match[2] || 5;
+    }
+    return msg.http(`http://pugme.herokuapp.com/bomb?count=${count}`).get()((err, res, body) => {
+      let i,
+        len,
+        pug,
+        ref,
+        results;
+      ref = JSON.parse(body).pugs;
+      results = [];
+      for (i = 0, len = ref.length; i < len; i++) {
```

Comment:
This whole `for` loop is confusing. 

All we need to do is send the pugs, right? So something like this would be much more acceptable

```javascript
const pugs = JSON.parse(body).pugs;

pugs.forEach(p => msg.send(pug));
```

Or just

`JSON.parse(body).pugs.forEach(p => msg.send(p));`

But if you're really into the one dot per line rule, you might be more inclined for the former which is also :100:.